masterlist.yaml

```
@@ -26659,8 +26659,8 @@ plugins:
     clean:
       - crc: 0x57A26548
         util: 'SSEEdit v4.0.4b'
-      - crc: 0x930C1CE3
-        util: 'SSEEdit v4.0.4b'
+      - crc: 0x28033725
+        util: 'SSEEdit v4.0.4c'
         
   - name: 'Dawn of RedBag''s Solitude'
```

I didn't catch this earlier, but, in #2614, you accidentally added some trailing whitespace. You can remove it here, or I can remove it in another commit, up to you.
```suggestion
        util: 'SSEEdit v4.0.4c'

  - name: 'Dawn of RedBag''s Solitude'
```